ably-ios/ARTProtocolMessage+Private.h

```diff
@@ -0,0 +1,23 @@
+//
+//  ARTProtocolMessage+Private.h
+//  ably-ios
+//
+//  Created by Toni Cárdenas on 26/01/2016.
+//  Copyright (c) 2014 Ably. All rights reserved.
+//
+
+ART_ASSUME_NONNULL_BEGIN
+
+@interface ARTProtocolMessage ()
+
+@property (art_nullable, readwrite, strong, nonatomic) NSString *clientId;
```

Comment:
Don't think a ProtocolMessage needs to have a clientId, http://docs.ably.io/client-lib-development-guide/features/#types